pkg/xen-tools/xen-start

```diff
@@ -19,7 +19,7 @@ xl create "$2" -p
 
 # we may need to wait for domain to come online for us to manipulate it (timing out in under 30 sec)
 for i in 1 2 3; do
-  ID=$(echo $(($(xl domid "$1") + 0))) ||:
+  ID=$(xl domid "$1" 2>/dev/null)
   sleep 8
```

Comment:
@rvs Don't understand why the old code did the sleep before the test/break. Do we need that 8 second delay?